### PR TITLE
[WIP] moving translation data tags clean-up functions to datasets_utils

### PR DESCRIPTION
**Pull request by parmeet**
This is follow up task for PR #1191 

Summary:
- Moved translational data tag clean-up functions to datasets_utils

### CODE REVIEW

File: torchtext/data/datasets_utils.py [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
-
+import codecs
+import xml.etree.ElementTree as ET
 """
 These functions and classes are meant solely for use in torchtext.datasets and not
 for public consumption yet.
 """
 
 
+def _clean_xml_file(f_xml):
```
Comment (zhangguanheng66) [inline torchtext/data/datasets_utils.py:17]: Are those building blocks general enough now to be placed in datasets_utils?

Comment (parmeet) [reply torchtext/data/datasets_utils.py:17]: It's a good question: I am not so sure about it right now. It is possible that they are too specific to IWSLT datasets. we should do some discussion on what we need to change among these two functions to make them more generic, or if not, what should be the better place to put this shared code.

Comment (zhangguanheng66) [reply torchtext/data/datasets_utils.py:17]: I'm not sure for now. One possible solution could be: add those utils functions in one dataset, and let other datasets import them from there whenever the datasets need.

**Comment by zhangguanheng66** [comment]
Merged by https://github.com/pytorch/text/pull/1224

**Pull request closed without merging**